### ISSUE TITLE
dayjs().toDate() get different value in browser and node

### ISSUE THREAD

**Issue by Joyeah** [iamkun/dayjs#673]
**Describe the bug**
dayjs().toDate() get different value in browser and node:
In browser, get the local time;
In node , get the GMT time.

**Expected behavior**
I hope to be a same value , such as a local time.

**Information**
 - Day.js Version : newest.
 - OS: Windows 10
 - Browser  chrome 76
 - Time zone: [e.g. GMT+08:00 ]

**Comment by xxyuk** [comment]
dayjs().toDate() will always return local time. Please check your node timezone setting.

**Comment by GitTom** [comment]
Yeah, this is the problem with most of the js Date API's: they just pickup Node's timezone (which picks up system timezone) and have no option set it manually to the user's timezone.  Intl.DateTimeFormat is better - allowing you to explicitly set the timezone for most functions - but keep in mind that all of these only have locale data for 'en-us' in Node versions < 13 so the date format will be wrong for all other countries anyway.
